src/MultiFormGoals/resources/views/progressbar.php

```diff
@@ -34,3 +38,9 @@
 		<?php endif; ?>
 	</div>
 </div>
+<script>
+	const container = document.getElementById('<?php echo $uniqid; ?>')
```

Comment:
Declaring this as a `const` means that the Progress Bar block can only be rendered once per page. If you add multiple Multi-Form Goal blocks to a page, you'll see this error in console: `Uncaught SyntaxError: Identifier 'container' has already been declared`

Reply:
Ah, yeah - that makes sense. I'll update the implementation.

Reply:
@henryholtgeerts I pushed an update to encapsulate the scripting.